api_types.ts

```diff
@@ -576,6 +576,12 @@ export interface FetchRequest {
    * You may inspect the `Location` header of the response to observe the indicated redirect URL.
    */
   ignoreRedirects?: boolean;
+  /**
+   * If true, Coda will cache the POST request and return the same response for subsequent requests.
+   * This is mainly used for POST requests that do not have side effects, such as querying a
```

Comment:
Nit, would strengthen this to say something like "This _must_ be used only for POST requests that do not have side effects..."